README.md

```diff
@@ -0,0 +1,54 @@
+
+# Mobile-menu
+
+
+## Built With
+- Html
+- CSS
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+html/css/github workflow knowledge
+
+### Setup
+Set up and configure git on your system.
+
+### Install
+```
+git clone https://github.com/Kalunge/mobile-menu.git
+
```

Comment:
[OPTIONAL] I noticed here that you can give a better explanation of how to use your repository. Think that giving very clear explanations will make look more professional your repository.  Why don't you try adding instructions on how to navigate to the just cloned repository and what file they should open?